api/v1alpha1/url_shortener.proto

```diff
@@ -38,4 +42,5 @@ message ShortURLsResponse {
 service URLShortening {
   // Creates short URLs from long URLs
   rpc ShortURLs(stream ShortURLsRequest) returns (stream ShortURLsResponse) {}
+  rpc BalanceURLs(ShortURLsRequests) returns (ShortURLsResponse) {}
```

Comment:
Also, the response should be another kind of message, I would create a new one called `BalanceURLsResponse` with the desired fields, like `string short_url = 1;`

Reply:
You are right, I've updated this PR @tembleking

Reply:
Thanks!